### DISCUSSION TITLE
Meget "følsom" genudgivelse

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#7] [Bugs]
Ikke en bug, men helt sikkert en uhensigtsmæssighed. Som måske endda let kan rettes. 

Alene det, at man klikker sig helt ind på en vagtdag udløser en genudgivelse. Vel at mærke uden, at man har ændret nogen værdier. Det er ofte en kilde til frustration, fordi events jo så hele tiden afpubliceres. 

Kan man ændre automatikken om afpublicering indtil gengodkendelse til, at den kun slår til, når værdier eller tekst ændres?

**Comment by antonlund18** [comment]
Jeg har lige prøvet at teste det, men jeg kunne ikke genskabe det. Jeg har dog oplevet at der kommer en boks frem, hvor der står 'Beskrivelse opdateret', selvom jeg ikke har opdateret beskrivelsen, så jeg tror helt bestemt der er et eller andet der bliver udløst ved en fejl.

Har du en helt konkret måde, hvorpå du kan fremprovokere det på? Fx hvordan klikker du dig ind på en vagtdag? Jeg har både prøvet fra eventkalenderen, og ved at gå ind på et event og gå til vagter.

**Comment by MartinQwist** [comment]
Denne havde jeg lige glemt. Men den er stadig sygt irriterende, så jeg undersøger og vender tilbage.
